core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20161202_add_json_attributevalue.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet id="add_json_entity_attribute" author="hussein" logicalFilePath="dummy">
+        <sql>
+            ALTER TABLE ENTITY_ATTRIBUTE ADD COLUMN value_json LONGTEXT;
+        </sql>
+        <sql>
+            ALTER TABLE ENTITY_ATTRIBUTE_SCRATCH ADD COLUMN value_json LONGTEXT;
+        </sql>
+    </changeSet>
+
+    <changeSet id="add_json_workspace_attribute" author="hussein" logicalFilePath="dummy">
+        <sql>
+            ALTER TABLE WORKSPACE_ATTRIBUTE ADD COLUMN value_json LONGTEXT;
+        </sql>
+        <sql>
+            ALTER TABLE WORKSPACE_ATTRIBUTE_SCRATCH ADD COLUMN value_json LONGTEXT;
+        </sql>
```

Comment:
I had assumed that one changeset = one transaction, so any failure would roll back the whole thing. How does it work, then?